src/test/java/org/broadinstitute/consent/http/resources/TDRResourceTest.java

```diff
@@ -98,4 +115,51 @@ public void testGetDatasetByIdentifier404() {
         assertEquals(404, r.getStatus());
     }
 
+    @Test
+    public void testCreateDraftDataAccessRequest() {
+
+        String identifiers = "DUOS-00001, DUOS-00002";
+
+        Dataset d1 = new Dataset();
+        d1.setDataSetId(1);
+
+        Dataset d2 = new Dataset();
+        d2.setDataSetId(2);
+
+        when(datasetService.findDatasetByIdentifier("DUOS-00001")).thenReturn(d1);
+        when(datasetService.findDatasetByIdentifier("DUOS-00002")).thenReturn(d2);
```

Comment:
You'll also need to mock out `darService.insertDraftDataAccessRequest`

Reply:
I was able to get this test to work with a few changes. 

1. Update the identifiers string to remove the extra space
2. Mock out `darService.insertDraftDataAccessRequest` to return a Dar with a darData
3. Mock out `uriInfo` since we're using that in the response. Need to do this twice, once for `path` and once for `build`.

Reply:
Mocking the darService and uriInfo helped get this test working.